proposal.md

```diff
@@ -1,31 +1,31 @@
 # Proposal
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a sports score
+My bot will respond by: looking at espn.com and giving that score back
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a joke 
+My bot will respond by: tell the user a random joke
 
-When a user says:
-My bot will respond by:
+When a user says: create an ai image
+My bot will respond by: make an ai image
```

Comment:
This is beyond the scope of the class. The goal of this project is to work with strings, not images. Let's come up with a different idea to replace this one.